packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -437,6 +437,10 @@ yarn workspace v1.22.19image` is a collocated image path, this entry will be the
     showLastUpdateTime: boolean;
     /** Whether to display the author who last updated the doc. */
     showLastUpdateAuthor: boolean;
+    /** Flattens folders so you can organise by year/month
+     * without it effecting the autogenerated slug
+     */
+    flattenFolders: boolean;
```

Comment:
To be honest reading that option name I have no idea at all what is the outcome of using.

If the goal is to have control over slug generation, we should rather expose a low-level callback like "generateSlug" instead?

Reply:
Yes some kind of way to programtically set the slug would be useful (think Wordpress options). Although for the purpose of cleaning up the blogs folder the existing functionality seems to be fine!